package.json

```diff
@@ -8,6 +8,10 @@
   "main": "dist/index.js",
   "module": "dist/index.es.js",
   "jsnext:main": "dist/index.es.js",
+  "files": [
+    "dist/*.js",
+    "dist/*.map"
+  ],
```

Comment:
This establishes [an allow list](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#files) for files to be included in the final package. I've verified that these files are all that are necessary to run `react-showtime`.